### PR TITLE
Adding the name to output string

### PR DESCRIPTION
**Pull request by choldgraf**
I noticed that a string isn't being properly populated w/ its corresponding value!

### CODE REVIEW

**Comment by betatim** [comment]
Nice work!

Could you also add a test to make sure this doesn't come back in the future? Maybe call `display_output` with a mocked `_get_notebook_outputs` checking that it raises/doesn't raise an exception.

**Comment by choldgraf** [comment]
hmmm - so I'm not sure the best way to test this. The exception itself is being raised fine, it is the *text* of the exception that's wrong. Anyone know of a way to check the text of an exception in tests (w/o building in a lot of complexity?)

**Comment by MSeal** [comment]
Something like

```
with self.assertRaisesMessage(PapermillException, '"Output Name foo is not available in this notebook."'):
  nb.display_output('foo')
```

**Comment by betatim** [comment]
Another way to do the same (for the pytest enthusiasts) https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions

```python
with pytest.raises(PapermillException) as excinfo:
    nb.display_output('foo')
assert 'Output Name foo is not available in this notebook.' in str(excinfo.value)
```

**Comment by choldgraf** [comment]
ok I think I figured it out - it was sort of a hybrid between both of your suggestions :-)

I didn't find a `assertRaisesMessage` method for the `unittest.TestCase` subclass, and it also didn't seem like pytest was being used so I didn't use that. In the end, it looks like `excinfo.exception` was the magical incantation

File: papermill/tests/test_api.py [hunk 1]
```diff
@@ -36,6 +36,10 @@ def test(self):
         )
         assert_frame_equal(nb.dataframe, expected_df)
 
+        with self.assertRaises(PapermillException) as excinfo:
```
Comment (mpacer) [inline papermill/tests/test_api.py]: `pytest.raises` rather than self.assertRaises?

File: papermill/tests/test_api.py [hunk 2]
```diff
@@ -36,6 +36,10 @@ def test(self):
...
+            nb.display_output('foo')
+        assert "Output Name 'foo' is not available in this notebook." in str(excinfo.exception)
```
Comment (mpacer) [inline papermill/tests/test_api.py]: Does excinfo.value not work per this [`pytest.raises`](https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions) example?

File: papermill/tests/test_api.py (hunk 1 above)
Comment (MSeal) [reply papermill/tests/test_api.py]: To be fair, this is a unittest object for self.Raises is perfectly valid.

**Comment by MSeal** [comment]
No need to spend a ton of cycles on testing a 1 line change for a message. If you want to try the pytest version awesome, otherwise I'm ok merging now for such a trivial change and we can do a test update pass in the future for these.

**Comment by choldgraf** [comment]
I'm happy to use pytest if it's better, I just didn't add to that test file since it wasn't being used anywhere else in there. Is papermill migrating away from unittest?

**Comment by MSeal** [comment]
Yes, gradually. I am probably the user keeping it in unittest the most by
not learning pytest equivilents everywhere :P. But in general pytest is
preferred here.

**Comment by betatim** [comment]
Travis is happy so I'm merging!

Thanks for the fix and humouring us with the testing extravaganza!

ps. the question for y'all to consider: who tests the tests?! 😱

**Comment by choldgraf** [comment]
cool - the latest push swapped out pytest for unittest 👍

**Pull request merged**